fastlane/env/user.env-example

```diff
@@ -1,3 +1,7 @@
+# Requires: 'repo'
 GITHUB_TOKEN=<GitHub access token>
 
+# Requires:
+#   Organizations: 'Automattic',
+#   REST Scopes: 'read_repos', 'write_repos'
```

Comment:
I don't see any `_repos` REST scopes on Buildkite. Could they have been `*_builds`?

<img width="716" alt="image" src="https://user-images.githubusercontent.com/1218433/196085426-a27555a0-d766-4307-9b5d-a4609d7134d5.png">

Also, I'm 99% confident the only scope we need is the write one, `write_builds`.

```suggestion
#   REST Scopes: 'write_builds'
```

We use the token in the `buildkite_trigger_build_action` from [release toolkit](https://github.com/wordpress-mobile/release-toolkit/, which uses the [`buildkit` gem](https://github.com/Shopify/buildkit internally. The gem [calls the `POST /builds` endpoint for us](https://github.com/Shopify/buildkit/blob/742e2e2ad8fc255f1e26143bf5502b017d754da9/lib/buildkit/client/builds.rb#L67-L85). The endpoint [documentation](https://buildkite.com/docs/apis/rest-api/builds#create-a-build) lists `write_builds` as the only required scope:

<img width="890" alt="image" src="https://user-images.githubusercontent.com/1218433/196085974-52aa9847-f9eb-4427-b046-f49213ef5bbc.png">

Reply:
Yes, I got my wires crossed there. Thanks! 🙇